src/main/java/com/gromoks/filestatistic/service/statistic/FileStatistic.java

```diff
@@ -0,0 +1,124 @@
+package com.gromoks.filestatistic.service.statistic;
+
+import com.gromoks.filestatistic.entity.FileInfo;
+import com.gromoks.filestatistic.entity.LineInfo;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class FileStatistic {
```

Comment:
(Optional) Move methods from FileStatistic class to FileProcessing class, to avoid support of extra dependancies